Documentation/Ctrl/Properties/Index.rst

```diff
@@ -1,51 +1,52 @@
-.. include:: /Includes.rst.txt
-.. _ctrl-reference:
+..  include:: /Includes.rst.txt
+..  _ctrl-reference:
 
 ==========
 Properties
 ==========
 
-.. toctree::
+..  toctree::
 
-   AdminOnly
-   Container
-   CopyAfterDuplFields
-   Crdate
-   DefaultSortby
-   Delete
-   DescriptionColumn
-   Editlock
-   Enablecolumns
-   Ext
-   FormattedLabelUserFunc
-   FormattedLabelUserFuncOptions
-   GroupName
-   HideAtCopy
-   HideTable
-   Iconfile
-   IsStatic
-   Label
-   LabelAlt
-   LabelAltForce
-   LabelUserfunc
-   LanguageField
-   OrigUid
-   PrependAtCopy
-   ReadOnly
-   RootLevel
-   SearchFields
-   Security
-   SeliconField
-   ShadowColumnsForNewPlaceholders
-   Sortby
-   Title
-   TranslationSource
-   TransOrigDiffSourceField
-   TransOrigPointerField
-   Tstamp
-   Type
-   TypeiconClasses
-   TypeiconColumn
-   UseColumnsForDefaultValues
-   VersioningWS
-   VersioningWSAlwaysAllowLiveEdit
+    AdminOnly
```

Comment:
Can this not be simply solved by a glob?

Reply:
Done